src/groovy/org/pih/warehouse/product/ProductSearchDto.groovy

```diff
@@ -45,7 +45,7 @@ class ProductSearchDto {
                 id                 : id,
                 productCode        : productCode,
                 name               : name,
-                translatedName     : translatedName,
+                displayName        : displayName,
```

Comment:
it was not straight forward to include whole `displayNames` here, since the data for `ProductSearch` is taken from pure SQL query.
It could be doable by fetching in the `ProductSearchDto` the `Product` by creating a method:

```groovy
Map getDisplayNames() {
  Product product = Product.get(id)
  return product.displayNames
}
```

But I think it would be an overkill and N+1 query if we wanted to fetch the whole `Product` inside the DTO that was supposed to shorten the `Product`....
thoughts?